docs/backend/portal-actions.md

```diff
@@ -11,3 +11,108 @@ myst:
+Therefore the i18n domain can be set here.
+
+The Action URL is the address of the page or feature that the action will access when it is clicked.
+This can be an internal page on your site or an external page on another website.
+
+The "Condition" controls whether to show the action or not.
+
+You can use expressions in the URL of an action to make the action more dynamic and flexible. Expressions are also used for the "Condition".
+
+The permissions chosen also control the visibility of an action.
+
+Visibility can be used to hide actions.
+This can be handy to disable default actions or temporarily disable actions.
+
+The "Position" is used to order action within their category and is numbered starting with "1".
+
+Together, these components make up a portal action in Plone.
+
+You can use the "Portal Actions" control panel to manage and organize your site's actions.
+
+## Action categories
+
+Several different action categories are used for different purposes.
+These categories include:
+
+Object Actions
+: `Actions that are available when viewing a specific content item.`
+
+Object Buttons
+: `Actions that are displayed as buttons when viewing a specific content item.`
+Portal Tabs
+: `Actions that are displayed as tabs on the top of the page.`
+
+Site Actions
+: `Actions that are available on every page of your site`
+
+User Actions
+: `Actions that are available to users when they are logged in to your site`
+
+Each of these action categories serves a different purpose and is intended for use in specific areas of the site.
+For example, object actions are intended for actions that are specific to a particular content item, while site actions are intended for actions that are available on every page of your site.
+
+## Expressions
+
+An expression is a piece of code that is evaluated at runtime and can be used to insert dynamic values into the URL or provide the decision for the condition.
+
+```{todo}
+Contribute to this documentation!
+A chapter about expressions is missing. As soon as it exists a link to this chapter needs to be placed here.
+See issue [Backend > missing chapter Expressions needs content](https://github.com/plone/documentation/issues/1370).
+```
+
+## Adding or editing portal actions
+
+To add or edit portal actions, you can use the Portal Actions control panel.
+To access this, log in to your Plone site as a user with the appropriate permissions, and then go to the {guilabel}`Site Setup` page.
+From there, click on the {guilabel}`Portal Actions` link in the {guilabel}`Plone Configuration` section.
+
+Once you are on the "Actions" control panel, you will see a list of all the existing portal actions for your site.
+To add a new action, click on the {guilabel}`Add new action` button at the top of the page.
+This will open a form where you can enter the details for your new action, such as the action name, URL, and any additional parameters.
```

Comment:
Add screenshot.

![Screen Shot 2022-12-13 at 12 34 27 AM](https://user-images.githubusercontent.com/102112/207265955-ab2cccfe-9671-4262-b807-2ba51a615c7f.png)

```suggestion
This will open a form where you can enter select its category and enter an arbitrary ID.
Click the {guilabel}`Add` button to add the action.
The new action should now appear in the actions listing under the selected category.
You can now edit its settings by clicking its {guilabel}`Edit` button.
```